exercises/Toluwalase_Agoro/ex01_basics/src/main.cpp

```diff
@@ -99,20 +99,23 @@ int main() {
     */
     
     holderString = getWord(fileToRead);
-    auto location = std::find_if(vectorOfCountsAndWords.begin(),vectorOfCountsAndWords.end(), myPredictor);
-    if (holderString.size() > 4 && location == vectorOfCountsAndWords.end()) {
-      vectorOfCountsAndWords.push_back(std::make_pair(1,holderString));
+    auto location = std::find_if(vectorOfCountsAndWords.begin(),vectorOfCountsAndWords.end(), myPredicator);
+    if (holderString.size() > 4){
```

Comment:
Please can you move the find_if line inside the if statement where you check the size of holderString. Don't think you need to do the call to find_if for words < 4